pkg/k8s/labels.go

```diff
@@ -183,6 +183,10 @@ const (
 	// MountPathBase is the base directory of the mount path.
 	MountPathBase = "/var/run/linkerd"
 
+	// MountPathServiceAccount is the default path where Kuberenetes stores
+	// the service account token
+	MountPathServiceAccount = "/var/run/secrets/kubernetes.io/serviceaccount"
```

Comment:
Are the service account volume mount changes in this diff related to the pod watch change that you're making, or are they maybe a result of a merge issue with the identity stuff?

Reply:
@klingerf it's related to the missing mount paths issue I referred to in this [comment](https://github.com/linkerd/linkerd2/pull/2472#issuecomment-472115321)

Reply:
Ah, I totally missed that, thanks. Will go back and take a look.